components/setting_item_max.jsx

```diff
@@ -267,7 +267,7 @@ export default class SettingItemMax extends React.PureComponent {
                     <ul
                         tabIndex='-1'
                         ref={this.settingList}
-                        role='presentation'
+                        role='list'
```

Comment:
@asaadmahmood Does this make sense to you?

Reply:
@devinbinnie Nops, I think we should avoid this change now not to accidentally change the readouts, we don't even need <ul>'s in there, and we can probably create a ticket to remove it. So we can avoid this for now.